.gitignore

```diff
@@ -68,3 +68,4 @@ GTAGS
 /.venv
 .tool-versions
 TAGS
+node_modules
```

Comment:
Totally minor but GitHub's diff is flagging that this file lost its trailing newline.